src/query-builder/select-query-builder.ts

```diff
@@ -1282,6 +1282,24 @@ export interface SelectQueryBuilder<DB, TB extends keyof DB, O>
    */
   clearSelect(): SelectQueryBuilder<DB, TB, {}>
 
+  /**
+   * Clears all `where` clauses from the query.
+   *
+   * ### Examples
+   *
+   * ```ts
+   * db.selectFrom('person')
+   *   .selectAll()
+   *   .where('first_name', '=', 'James')
+   *   .clearWhere()
+   * ```
+   *
+   * The generated SQL(PostgreSQL):
+   *
+   * ```sql
+   * select * from "person"
+   * ```
+   */
```

Comment:
These clearWhere comments were omitted on purpose. The comment is copied from the `WhereInterface` during compilation.